plugins/woocommerce-blocks/assets/js/base/stores/add-to-cart.ts

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+import { store } from '@woocommerce/interactivity';
+import type { store as StoreType } from '@wordpress/interactivity'; // Todo: remove once we import from `@wordpress/interactivity`.
+
+export interface Store {
+	state: {
+		restUrl: string;
+		nonce: string;
+		cart: {
+			items: {
+				key?: string;
+				id: number;
+				quantity: number;
+			}[];
+		};
+	};
+	actions: {
+		addToCart: ( id: number, quantity: number ) => void;
+		refreshCart: () => void;
+	};
+}
+
+let pendingRefresh = false;
+
+// Todo: Remove the type cast once we import from `@wordpress/interactivity`.
+export const { state, actions } = ( store as typeof StoreType )< Store >(
+	'woocommerce',
+	{
+		actions: {
+			// Question: Should we name this `addCartItem` to match the Store API naming?
+			// Question: Should we use regular arguments or wrap them in an object?
```

Comment:
Question: Should we use regular arguments or wrap them in an object?

It would be good to maintain consistency among all actions and use either arguments or objects in all of them.

`addToCart( id, quantity )` vs `addToCart( { id, quantity } )`

To do this, it would be good to take a look at the Store API to see what other actions we need to add and see if in some of them it makes more sense to use objects.

Reply:
As there are parts of the Store API that have a few arguments, including optional arguments ([for example](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/src/StoreApi/docs/checkout.md#process-order-and-payment)), I believe that using objects as the standard is going to lead to a better DX.